docs/guides/browser-extension/install-browser-extension.md

```diff
@@ -42,60 +42,25 @@ Projects built by the LUKSO community:
 
 Click on link below to download the extension, based on the browser to which you want to install it.
 
-### :inbox_tray: **[Download link (v1.3.3-MV2)](https://storage.googleapis.com/up-browser-extension/universalprofile-extension-v1.3.3-mv2.zip) (All other browsers)**
+### :inbox_tray: **[Download from Chrome Web Store](https://chrome.google.com/webstore/detail/universal-profiles/abpickdkkbnbcoepogfhkhennhfhehfn) (Chrome / Brave)**
 
-### :inbox_tray: **[Download link (v1.3.3-MV3)](https://storage.googleapis.com/up-browser-extension/universalprofile-extension-v1.3.3-mv3.zip) (Chrome)**
+
+### :inbox_tray: **[Download Archive](https://storage.googleapis.com/up-browser-extension/universalprofile-extension-v1.4.0-mv2.zip) (All other browsers)**
```

Comment:
@asciiman @Hugoo @biancabuzea200 some questions

So this link is always pointing to a specific version of the extension on the chrome store?

**question 1:** If you release a new version of the extension (_e.g: 1.4.18 is the latest with several bug fixes_), how can we ensure that the user will download always the latest version when clicking on this button?

**question 2:** Is there not something like `latest`?

**question 3:** also, shouldn't it be mv3 favoured instead of mv2?

Reply:
The code on line 48 is not for the chrome store. This is the hardcoded link to google storage for non-chrome browsers. The link to the chrome store is on line 45, and this is always "latest".

Reply:
That means that for non-chrome browsers, we cannot have a "latest" link and will always have to update the link manually?